src/components/cards/ListViewTaskCard.tsx

```diff
@@ -2,15 +2,34 @@
 
 import { Avatar, Box, Stack, Typography } from '@mui/material'
 import { AppMargin, SizeofAppMargin } from '@/hoc/AppMargin'
-import { TaskResponse } from '@/types/dto/tasks.dto'
+import { TaskResponse, UpdateTaskRequest } from '@/types/dto/tasks.dto'
 import { useSelector } from 'react-redux'
 import { selectTaskBoard } from '@/redux/features/taskBoardSlice'
+import { SelectorType } from '../inputs/Selector'
+import { useHandleSelectorComponent } from '@/hooks/useHandleSelectorComponent'
```

Comment:
Please prefer absolute / aliased imports instead of relative

Reply:
Updated in ff6350b62e855d120d47f943639badd245c0f85e commit.
However, there can be other multiple places where there is relative imports being used for imports from upto level 2 i.e. `..`. This should be tackled by configuring eslint and prettier in a separate PR.